### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by hkollmann**
I found a lot of files where the version string was 5.1.0. So i run bumpqxversion.py to change it in all files to 6.0.0-alpha.
I also add some licence fields in package.json.
And third i moved the grunt stuff in /package.json to devDependencies

### CODE REVIEW

**Review by cboulanger — DISMISSED** [review]

**Review by level420 — DISMISSED** [review]

File: documentation/manual/source/conf.py [hunk 1]
```diff
@@ -56,17 +56,17 @@
 # built documents.
 #
 # The short X.Y version.
-version = '6.0.0'
+version = '6.0.0-alpha'
 # The full version, including alpha/beta/rc tags.
 release = '6.0.0-alpha'
 # The current git branch (used for github links)
-git_branch = "master"
+git_branch = "bump_version"
```
Comment (cajus) [inline documentation/manual/source/conf.py]: Is that correct?

File: tool/admin/release/index.html [hunk 2]
```diff
@@ -209,42 +209,42 @@ <h2>Website Tests</h2>
           <h2>Website Lib</h2>
```
Comment (cajus) [inline tool/admin/release/index.html]: I'd vote for dropping the admin stuff and most parts of the tech documentation before 6.0. It does not get deployed and the technical information in there is outdated anyways.

Comment (johnspackman) [reply tool/admin/release/index.html]: :+1

**Review by johnspackman — DISMISSED** [review]

File: tool/admin/release/index.html (hunk 2 above)
Comment (hkollmann) [reply tool/admin/release/index.html]: +1 but we should have some technical information somewhere. It was painfull to figure this out. And it‘s not very obvious how you can make a lokal build to test the changes in the build toolchain

File: documentation/manual/source/conf.py (hunk 1 above)
Comment (hkollmann) [reply documentation/manual/source/conf.py]: changed it back

File: tool/admin/release/index.html (hunk 2 above)
Comment (hkollmann) [reply tool/admin/release/index.html]: But this is another sory.

**Review by level420 — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Review by derrell — APPROVED** [review]

**Review by cboulanger — APPROVED** [review]

**Comment by hkollmann** [comment]
It´s documentation. So 2 approvals are enough.

**Comment by level420** [comment]
@hkollmann yes it is a doc only PR, but from our dev rules you’re not allowed to merge your own PR. For my PRs I’m allways asking in the core team for someone who is willing to merge.

**Comment by hkollmann** [comment]
Sorry for this next i will do so!

**Comment by level420** [comment]
@hkollmann you're right! it was never in the docs. Maybe we only talked about the restriction and we had it in practice without writing it down. Sorry about this.

**Pull request merged**